boards/nxp/imx93_evk/imx93_evk_mimx9352_a55_smp_defconfig

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
How about include imx93_evk_mimx9352_a55_defconfig and added SMP specified configure items?

Reply:
AFAIK, it doesn't support 'include' in defconfig.